app/styles/modules/_m-orange-bar.scss

```diff
@@ -51,10 +51,13 @@
       padding-left: 0.75rem;
     }
   }
-  li:hover, li:active, a:hover, a:active, .dropdown:hover .dropbtn {
+  li:hover, li:active, .dropdown:hover .dropbtn {
```

Comment:
Removing this is causing the Modify Tables secondary button in the Explorer to not have a white background on hover: 
![image](https://user-images.githubusercontent.com/3311663/125853205-43fd3e2b-38c5-42f9-ad24-0e4c5663f375.png)